devel/git-town/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/git-town/git-town 7.6.0 v
+go.setup            github.com/git-town/git-town 7.8.0 v
 revision            1
```

Comment:
`revision` should be reset to 0 whenever the version number gets bumped.

Please keep this change in the same one commit.

Reply:
I had actually meant resetting the value to 0 specifically, instead of deleting the entire line, but that's fine.